op3_ball_detector/include/op3_ball_detector/ball_detector_config.h

```diff
@@ -52,10 +53,13 @@ const int X_MIN_DEFAULT = 0;
 const int X_MAX_DEFAULT = 4000;
 const double LIGHT_SLOPE_DEFAULT = 1;
 const double LIGHT_CONSTANT_DEFAULT = 0;
-const int NUM_INTERVALS = 10; // test at 10
+const int NUM_INTERVALS = 20; // test at 10
 const double RANDOM_SAMPLE_CHANCE = 0.05;
-const int DETECTION_REWARD = 4;
-const int DETECTION_PENALTY = -1;
+const double DETECTION_REWARD_SCALAR = 4.0;
+const int DETECTION_PENALTY_SCALAR = -4.0;
```

Comment:
DETECTION_PENALTY_SCALAR should also be a const double

Reply:
Changed in b292ec1